app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Person has to be vaccinated")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("The vaccine is expired")
+        elif visitor["wearing_a_mask"] is False:
```

Comment:
There is a code efficiency issue here. Instead of checking if 'wearing_a_mask' is False, you can simply write `if not visitor['wearing_a_mask']:`. This is more pythonic and efficient.